clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -275,6 +276,14 @@ public HostLevelIngestionStats(
         new Max(),
         TehutiUtils.getPercentileStat(getName() + AbstractVeniceStats.DELIMITER + valueSizeSensorName));
 
+    String assembledValueSizeSensorName = "assembled_record_value_size_in_bytes";
+    this.assembledValueSizeSensor = registerSensor(
+        assembledValueSizeSensorName,
+        new Avg(),
+        new Min(),
+        new Max(),
+        TehutiUtils.getPercentileStat(getName() + AbstractVeniceStats.DELIMITER + assembledValueSizeSensorName));
```

Comment:
I remember some conversation about moving away from _Tehuti_ to _Open Telemetry_. Can you confirm if parts of code base already use _Open Telemetry_? In which case, you can modify this to use _Open Telemetry_.

Reply:
It would be nice to use OTel, but I think that's still an ongoing migration. I ran a quick search through the codebase, but I wasn't able to find anything using OTel.